### DISCUSSION TITLE
Pen only feature?

### DISCUSSION THREAD

**Discussion by GWMCwing** [geovens/gInk#102] [Ideas]
Maybe there can be an option that only the physical pen, like Wacom pen, input is allowed to be draw on the draw, and i can continue using my mouse to navigate on the screen?
I am using this app for quick annotation, but that is a bit uncomfortable that i can't use the mouse while drawing on the screen, like the original whiteboard in win store

**Comment by YouFoundJK** [comment]
Hey, gInk already have a very similar feature. Well, its not auto detection, but you could toggle back and forth between cursor and pen by using default of a simpler hotkey. These are my hotkeys:
![image](https://user-images.githubusercontent.com/72195985/125041447-664e2880-e0b6-11eb-9f5f-d105e54df367.png)

Once the gInk app is started you just have to press the hotkey to toggle to pen and press that again if you want to move back to mouse cursor. Further, you could define this hotkey in one of your Wacom tablet buttons and switch back and forth pretty quickly. In do exactly this, not in Wacom but with Xp-Pen tablet.
